components/MapSection/MapSection.tsx

```diff
@@ -69,10 +70,18 @@ const MapSection: FunctionComponent<MapSectionProps> = ({ onChangeLocation, ...p
   const onOptionsChange = (optns: SpotlightOptions): void => {
     setQuery(optns, activeLocation && [activeLocation]);
     setOptions(optns);
+    addEvent('mapsectionOptionsChanged', {
+      topic: optns.theme?.name,
+      indicator: optns.indicator?.name,
+      year: optns.year
+    });
   };
   const onSelectLocation = (location?: SpotlightLocation): void => {
     setQuery(options, location && [location]);
     setActiveLocation(location);
+    addEvent('locationChanged', {
+      locationName: location?.name.toUpperCase()
```

Comment:
Shouldn't this at least default to the country name for when there's no location?

Reply:
Write it on one line ... sure it'll fit.

Reply:
Updated to default to country name when there's no location which makes it too long for one line.